docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,197 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Definitions
 
-- _**Term**: Definition._
+- **Image Compatibility specification**: A specification that describes the container image requirements for the host operating system.
 
-## User Stories
+## Member-submitted user stories (Distilled and organized by roles)
 
-1. As a _role_, I want to _task_.
+The user stories prepared based on the [Member-submitted user stories (raw)](#member-submitted-user-stories-raw).
+
+All the user stories have been validated by the working group memebers, however, just because something is on the list does not mean it MUST be implemented and included in proposals.
+
+Stated more clearly, the list is a suggestion of use cases that proposals SHOULD consider.
+
+### Spec Personas and Roles
+
+The following is a list of personas and roles used to describe different contexts from a user story perspective:
+
+- **Image Author**: person that builds and maintains the image artifact (container, compatibility specification).
+- **Domain Architect**: person that provides the expertise for how images - should be described in the compatibility specification.
+- **Tool Writer**: person that builds tools to manage images on registries and other locations.
+- **System Runtime Administrator**: person who is responsible for maintaining the container runtime environment on which OCI containers (e.g. in HPC, just the Syadmin, person deploying cluster or individual runtime nodes)
+- **Deployment Engineer**: person who owns managing the application deployment on to a container runtime cluster/environment/host.
+- **Registry Maintainer**: person maintaining the registry (e.g., Docker Hub, Harbor).
+- **OCI Specification Maintainer**: someone from the open containers initiative (aka us).
+- **Security Administrator**: person responsible for securing the environment.
+
```

Comment:
I would prefer to keep it as we have in the document https://docs.google.com/document/d/1ULXHY0pdiLBlGZPN2Gj54XSvAo--UXKYr4opTrWSFMs/edit#heading=h.d3gle52ih5ke

Reply:
Oh, wait a sec. I had to read your comment again. That makes sense.

Reply:
I preserved the role names and adjusted to action verb style.
Since I am not a native speaker... if you have a better idea or you find something wrong then please propose :)

Reply:
It's great now! The succinctness is :kissing_heart: